packages/nextjs/components/Footer.tsx

```diff
@@ -23,7 +23,7 @@ export const Footer = () => {
                 <span>{ethPrice}</span>
               </div>
             )}
-            {scaffoldConfig.targetNetwork.id === hardhat.id && <Faucet />}
+            {Number(scaffoldConfig.targetNetwork.id) === hardhat.id && <Faucet />}
```

Comment:
Had to use this hack because it was giving below error : 
![Screenshot 2023-03-20 at 12 44 12 AM](https://user-images.githubusercontent.com/80153681/226203381-9fbce502-7136-4986-af11-627e816bd557.jpg)

Other option will be using `// @ts-ignore`

Reply:
I think rather than solving it with javascript, you can solve it by simply casting it to `number`, this should widen its type:

```typescript
(scaffoldConfig.targetNetwork.id as number) === hardhat.id
```

Reply:
it works without `Number` or `as`
`scaffoldConfig.targetNetwork.id === hardhat.id && <Faucet />`